packages/clients/package.json

```diff
@@ -64,7 +64,7 @@
     "superstruct": "^1.0.3"
   },
   "devDependencies": {
-    "@aptos-labs/ts-sdk": ">=1.13.3",
+    "@aptos-labs/ts-sdk": ">=1.22.2",
```

Comment:
Are we using any new features from this version?

Reply:
No new features, but wanted to keep versioning up to date. Do you recommend keeping the previous version?

Reply:
Here it's not about "keeping it up to date". It's specifying a minimum bar for the [peer dependency](https://nodejs.org/en/blog/npm/peer-dependencies). The lower the bar, the more compatible this library is in dependent projects.

> One piece of advice: peer dependency requirements, unlike those for regular dependencies, should be lenient. You should not lock your peer dependencies down to specific patch versions. It would be really annoying if one Chai plugin peer-depended on Chai 1.4.1, while another depended on Chai 1.5.0, simply because the authors were lazy and didn't spend the time figuring out the actual minimum version of Chai they are compatible with.